preset: switch VP9 MKV presets to Constant Quality mode

marcosfrm opened the pull request:
4137019c5771aa061419893e6f5747649e408dd4 enabled VPX_Q (Constant Quality mode) for libvpx-vp9. This commit switches VP9 MKV presets to utilize this mode.

Additionally, libvpx demonstrates improved results with 2-pass encoding, which is now supported with CQ thanks to the changes from #5899.

Some metrics using derf's samples:

big_buck_bunny_480p24.y4m (VP9 MKV 480p30)

|            | Current preset | CQ 28 2-pass | CQ 30 2-pass |
|------------|----------------|--------------|--------------|
| VMAF Score | 97.947086      | 96.799294    | 96.497227    |
| File size  | 93,316,706     | 59,412,383   | 53,409,106   |

city_4cif.y4m (VP9 MKV 480p30)

|            | Current preset | CQ 28 2-pass | CQ 30 2-pass |
|------------|----------------|--------------|--------------|
| VMAF Score | 94.675175      | 96.212573    | 95.199821    |
| File size  | 2,755,841      | 4,046,760    | 3,478,329    |

ice_4cif.y4m (VP9 MKV 576p25)

|            | Current preset | CQ 28 2-pass | CQ 30 2-pass |
|------------|----------------|--------------|--------------|
| VMAF Score | 98.902469      | 98.298897    | 97.920450    |
| File size  | 1,303,578      | 1,009,026    | 915,173      |

720p50_parkrun_ter.y4m (VP9 MKV 720p30)

|            | Current preset | CQ 28 2-pass | CQ 30 2-pass |
|------------|----------------|--------------|--------------|
| VMAF Score | 97.203104      | 97.768514    | 97.552664    |
| File size  | 21,029,808     | 29,284,211   | 25,703,744   |

sunflower_1080p25.y4m (VP9 MKV 1080p30)

|            | Current preset | CQ 28 2-pass | CQ 30 2-pass |
|------------|----------------|--------------|--------------|
| VMAF Score | 99.059100      | 98.497331    | 98.257833    |
| File size  | 11,201,138     | 8,266,523    | 6,997,762    |

Netflix_Crosswalk_4096x2160_60fps_10bit_420.y4m (VP9 MKV 2160p60 4K)

|            | Current preset | CQ 28 2-pass | CQ 30 2-pass |
|------------|----------------|--------------|--------------|
| VMAF Score | 93.195042      | 94.927541    | 94.224311    |
| File size  | 8,549,789      | 15,747,020   | 13,683,317   |

***

Remarks/Questions:

- Should I increment the presets version?
- CQ 30 2-pass seems to perform decently overall. More testing is welcome.

**Tested on:**

- [x] Windows 10+  (via MinGW)
- [ ] macOS 10.13+
- [ ] Ubuntu Linux

bradleysepos reviewed (approved):
Seems fine to me. No doubt these ancient settings no longer apply to modern libvpx.

Make sure to bump `VersionMajor` in `preset/preset_builtin.list` so the graphical interfaces pick up the change without the need for manual action by the user.

marcosfrm committed:
preset: switch VP9 MKV presets to Constant Quality mode

4137019c5771aa061419893e6f5747649e408dd4 enabled VPX_Q (Constant Quality mode) for libvpx-vp9. This commit switches VP9 MKV presets to utilize this mode.

Additionally, libvpx demonstrates improved encoding results with 2-pass encoding, which is now supported with CQ thanks to the changes from #5899.

Some metrics using derf's samples:

big_buck_bunny_480p24.y4m (VP9 MKV 480p30)

|            | Current preset | CQ 28 2-pass | CQ 30 2-pass |
|------------|----------------|--------------|--------------|
| VMAF Score | 97.947086      | 96.799294    | 96.497227    |
| File size  | 93,316,706     | 59,412,383   | 53,409,106   |

city_4cif.y4m (VP9 MKV 480p30)

|            | Current preset | CQ 28 2-pass | CQ 30 2-pass |
|------------|----------------|--------------|--------------|
| VMAF Score | 94.675175      | 96.212573    | 95.199821    |
| File size  | 2,755,841      | 4,046,760    | 3,478,329    |

ice_4cif.y4m (VP9 MKV 576p25)

|            | Current preset | CQ 28 2-pass | CQ 30 2-pass |
|------------|----------------|--------------|--------------|
| VMAF Score | 98.902469      | 98.298897    | 97.920450    |
| File size  | 1,303,578      | 1,009,026    | 915,173      |

720p50_parkrun_ter.y4m (VP9 MKV 720p30)

|            | Current preset | CQ 28 2-pass | CQ 30 2-pass |
|------------|----------------|--------------|--------------|
| VMAF Score | 97.203104      | 97.768514    | 97.552664    |
| File size  | 21,029,808     | 29,284,211   | 25,703,744   |

sunflower_1080p25.y4m (VP9 MKV 1080p30)

|            | Current preset | CQ 28 2-pass | CQ 30 2-pass |
|------------|----------------|--------------|--------------|
| VMAF Score | 99.059100      | 98.497331    | 98.257833    |
| File size  | 11,201,138     | 8,266,523    | 6,997,762    |

rush_hour_1080p25.y4m (VP9 MKV 1080p30)

|            | Current preset | CQ 28 2-pass | CQ 30 2-pass |
|------------|----------------|--------------|--------------|
| VMAF Score | 95.683483      | 94.849661    | 94.016348    |
| File size  | 11,251,468     | 9,174,195    | 8,059,276    |

Netflix_Crosswalk_4096x2160_60fps_10bit_420.y4m (VP9 MKV 2160p60 4K)

|            | Current preset | CQ 28 2-pass | CQ 30 2-pass |
|------------|----------------|--------------|--------------|
| VMAF Score | 93.195042      | 94.927541    | 94.224311    |
| File size  | 8,549,789      | 15,747,020   | 13,683,317   |

CQ 30 2-pass seems to perform decently overall.

marcosfrm committed:
cli: fix typo in help message

marcosfrm commented:
Done. Added a cli typo fix.

bradleysepos commented:
LGTM
